build/org.eclipse.pde.build.tests/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.pde</groupId>
+		<artifactId>eclipse.pde.build</artifactId>
+		<version>4.31.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.eclipse.pde.build.tests</artifactId>
+	<version>1.4.300-SNAPSHOT</version>
+	<packaging>eclipse-test-plugin</packaging>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<configuration>
+					<dependency-resolution>
+						<extraRequirements>
+							<requirement>
+								<!-- The product -->
+								<type>p2-installable-unit</type>
+								<id>org.eclipse.sdk.ide</id>
+								<versionRange>0.0.0</versionRange>
+							</requirement>
+						</extraRequirements>
+					</dependency-resolution>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-surefire-plugin</artifactId>
+				<configuration>
+					<product>org.eclipse.sdk.ide</product>
+					<testRuntime>p2Installed</testRuntime>
+					<testClass>org.eclipse.pde.build.tests.PDEBuildTestSuite</testClass>
```

Comment:
Am I right, that this testClass is used for the default `test` goal execution of the `tycho-surefire-plugin` for `eclipse-test-plugins` and the overwritten value in the separate `p2-tests` execution refelects the second execution defined in this projects test.xml?

Reply:
Yes